src/main/resources/swagger/api-docs.yaml

```diff
@@ -524,6 +524,13 @@ paths:
           required: true
           type: string
           in: path
+        - name: workspace
+          description: Cloned workspace information
+          required: true
+          type: string
+          in: body
+          schema:
+            $ref: WorkspaceIngest
```

Comment:
I'm not clear on why we need to have a full WorkspaceIngest as the body of the post. If we're cloning a ws, there are only two things that can happen wrt realms. 
1. The source ws is already protected so the clone has to be as well. 
2. The option to make the clone protected, regardless of the source ws. 
Isn't it sufficient to just add a single parameter, something like `isProtected` to the POST?